src/openapi/streaming/subscription.js

```diff
@@ -1,4 +1,4 @@
-/* eslint max-lines: ["error", 600] */
+/* eslint max-lines: ["error", 650] */

```

Comment:
Minimal amount of new code was added to subscription. Actually no new logic, only invocation of encapsulated serializers and comments.

New line count is 623. So if I would need to keep max 600, refactoring of subscription.js outside my changes is required.